sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -280,7 +283,7 @@ scatter(Tx *p, Toffset offset, simd<Tx, N> vals, simd_mask<N> mask = 1) {
 template <typename Tx, int N,
           typename Flags = overaligned_tag<detail::OperandSize::OWORD>>
 __ESIMD_API std::enable_if_t<is_simd_flag_type_v<Flags>, simd<Tx, N>>
-block_load(const Tx *addr, Flags = {}) {
+block_load(const Tx *addr, Flags) {
```

Comment:
It is not a breaking change. And this change is also required because now the defaultly chosen function for block_load(ptr) call must be not this one, but block_load(ptr, props={}).